participants/Deepcoin.md

```diff
@@ -0,0 +1,64 @@
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+(0) Project Name and Team Slack Handles
+
+	Project Name: Deepcoin
+	Team Slack Handles ：motingzheng@deepcoin.com
+	Submission ID: 673373055
+	
+(1) Brief Project Description
+
+    Deepcoin is a global digital asset trading platform with all-inclusive financial services. It was founded in 2018, established in seychelles and headquartered in Singapore. As our name called, in order to provide Deep liquidity, we hold positions on major exchanges around the world and transfer liquidity between them, allowing our users to enjoy good enough trading depth, especially in extreme markets. One of the biggest problems is the risk of delay in transferring time on the chain and the risk of holding positions on various exchanges, particularly Bitcoin. Therefore, we use artificial intelligence algorithms to make some predictions about the price trends of each exchange and the distribution of liquidity across the network
+
+(2) Link to Project UI:
+
+    https://www.deepcoin.com 
+	
+(3) What does your application/UI do?
+
+	We need to collect and store large amounts of data and deploy artificial intelligence analysis. Here are some examples：
+	The clustering algorithm is used to analyze the transfer data on the whole chain and construct the transfer map on the chain, such as identifying large accounts which disguised as a large number of retail addresses and analyzing their asset movements. 
+	Using the Hidden Markov Model, the price and volume data of the exchange is analyzed to predict the large fluctuation rate and direction change. 
+	We also analyze the order book data of each exchange ,and deploy  Reinforcement Learning through a large number of order-making experiments to analyze and identify their market-maker’s strategies and liquidity movements.
+
+(4) If your project is using a curated dataset, which one are you using?
+
+	Data are obtained from many public chains and analyzed and predicted
+
+(5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+	Analyze the whole network of some reliable chain trading information
+
+(6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+    About 100 TB
+	
+(7) How are you structuring the data?
+
+	Data parsed from different public chains will be structured and saved.
+	
+(8) What pre-processing are you doing before ingesting the data?
+
+	After the data is obtained, it will be classified and collected by our customized program, and the data with reference value will be extracted and saved
+
+(9) What tech stack will you use to store the data?
+
+	Data centers and object storage on the cloud.
```

Comment:
Can you specify whether you'll be using Lotus or Powergate for storage?  How will you enable retreival?